python/typedb/driver.py

```diff
@@ -77,5 +78,7 @@ def cloud_driver(addresses: Union[Iterable[str], str], credential: TypeDBCredent
         """
         if isinstance(addresses, str):
             return _Driver([addresses], credential)
+        elif isinstance(addresses, Mapping):
+            return _Driver(dict(addresses), credential)
```

Comment:
A `Mapping` is an `Iterable`, but not vice versa, so this check must come first.